### ISSUE TITLE
Improve message when logs are inquired before action is run

### ISSUE THREAD

**Issue by ioana-blue** [openwhisk/openwhisk#373]
I noticed the following behavior: I invoke an action (async) and immediately I ask for the logs. I get something like this:
```
./wsk action invoke slack-channels-info
ok: invoked slack-channels-info with id cb5930907976445aaebe4a5fd661d304
ioana@Ioanas-MBP:bin [ioana-slackbot]$ ./wsk activation logs cb5930907976445aaebe4a5fd661d304
error: The requested resource does not exist. (code 353)
```
The message is misleading, there is no error, it's just that the action was not run yet (it's pending). We should fix this message.

**Comment by rabbah** [comment]
The implementation does not create an activation until the activation is completed. So a GET for "valid" activation id which is received as part of a non-blocking invoke will return `404` (Not Found) until the activation completes. The controller will need to create an activation id with a `status` marked `pending`.

**Comment by jthomas** [comment]
I've also run into issues due to this design. There's a couple of use-cases where I'd like to be able to retrieve an activation record before it has finished....

- Monitoring activation from non-blocking invoke. Currently you'd have to poll the record waiting for the 404 to disappear but it's a bit error prone in case the activation id was wrong. 

- Checking for in-flight activations. I have an action connected to a timer that varies in execution duration. If the action takes too long to execute, the next trigger will fire before it has finished. If I could check for in-flight activations, the next action could exit instead of running.

**Comment by shakshisinghai** [comment]
Yes, I am also facing the same issue. This is really confusing. 

![image](https://user-images.githubusercontent.com/37688219/202828364-1e0a3bd3-e01c-405d-9520-99b2d62ffd83.png)
